services/ui-src/package.json

```diff
@@ -34,6 +34,7 @@
     "react-query": "^3.34.5",
     "react-router-dom": "6.0.2",
     "react-scripts": "^5.0.0",
+    "react-scroll": "^1.8.7",
```

Comment:
Do we need this additional library? Looking at [the react router guide](https://v5.reactrouter.com/web/guides/scroll-restoration), it seems like we should be able to handle this with existing dom methods.

Reply:
This is something we discussed a bit, but `react-scroll` allows you greater control over the scroll behavior ([options](https://github.com/fisshy/react-scroll#propsoptions)) and will be useful down the line when design wants to target specific elements as scroll destinations and handle fancy scrolling behaviors in plan-specific drawers within the form.

Reply:
(See Brax's comments below):

We did originally have it using window.scrollTo, like react router, but we used this because it more easily exposes options for animations, duration, delay, etc. If you think there's never a need to animate, I'll change it back. Just did this because it'd be easier for form stuff in the future (related to your question below: we may not need that anyway)

Reply:
Could we defer bringing in the additional dependency until its features are needed? I think the dom's scrollTo on both the window and element objects (along with the optional `smooth` behavior) covers everything that is currently needed.